test/test_distance.py

```diff
@@ -286,3 +297,91 @@ def test_should_get_distinct_results_for_different_ellipsoids(self):
+           8935244.5604818305],
+          [55.52454, 106.05087, 22.020059880982801,
+           77.03196, 197.18234, 109.112041110671519,
+           4105086.1713924406],
+          [-21.97856, 142.59065, -32.44456876433189,
+           41.84138, 98.56635, -41.84359951440466,
+           8394328.894657671],
+          [-66.99028, 112.2363, 173.73491240878403,
+           -12.70631, 285.90344, 2.512956620913668,
+           11150344.2312080241],
+          [-17.42761, 173.34268, -159.033557661192928,
+           -15.84784, 5.93557, -20.787484651536988,
+           16076603.1631180673],
+          [32.84994, 48.28919, 150.492927788121982,
+           -56.28556, 202.29132, 48.113449399816759,
+           16727068.9438164461],
+          [6.96833, 52.74123, 92.581585386317712,
+           -7.39675, 206.17291, 90.721692165923907,
+           17102477.2496958388],
+          [-50.56724, -16.30485, -105.439679907590164,
+           -33.56571, -94.97412, -47.348547835650331,
+           6455670.5118668696],
+          [-58.93002, -8.90775, 140.965397902500679,
+           -8.91104, 133.13503, 19.255429433416599,
+           11756066.0219864627],
+          [-68.82867, -74.28391, 93.774347763114881,
+           -50.63005, -8.36685, 34.65564085411343,
+           3956936.926063544],
+          [-10.62672, -32.0898, -86.426713286747751,
+           5.883, -134.31681, -80.473780971034875,
+           11470869.3864563009],
+          [-21.76221, 166.90563, 29.319421206936428,
+           48.72884, 213.97627, 43.508671946410168,
+           9098627.3986554915],
+          [-19.79938, -174.47484, 71.167275780171533,
+           -11.99349, -154.35109, 65.589099775199228,
+           2319004.8601169389],
+          [-11.95887, -116.94513, 92.712619830452549,
+           4.57352, 7.16501, 78.64960934409585,
+           13834722.5801401374],
+          [-87.85331, 85.66836, -65.120313040242748,
+           66.48646, 16.09921, -4.888658719272296,
+           17286615.3147144645],
+          [1.74708, 128.32011, -101.584843631173858,
+           -11.16617, 11.87109, -86.325793296437476,
+           12942901.1241347408],
+          [-25.72959, -144.90758, -153.647468693117198,
+           -57.70581, -269.17879, -48.343983158876487,
+           9413446.7452453107],
+          [-41.22777, 122.32875, 14.285113402275739,
+           -7.57291, 130.37946, 10.805303085187369,
+           3812686.035106021],
+          [11.01307, 138.25278, 79.43682622782374,
+           6.62726, 247.05981, 103.708090215522657,
+           11911190.819018408],
+          [-29.47124, 95.14681, -163.779130441688382,
+           -27.46601, -69.15955, -15.909335945554969,
+           13487015.8381145492]]
+        d = self.cls(ellipsoid = 'WGS-84')
+        km = 1000
+        for l in testcases:
+            (lat1, lon1, azi1, lat2, lon2, azi2, s12) = l
+            # lon2 is the UNROLLed longitude, so normalize
+            lon2 = (lon2 + 360 if lon2 < -180 else
```

Comment:
Will fix (normalization isn't necessary)

Reply:
No, I *do* need to normalize.  I retain my original idiom since this puts the longitude in (-180,180] whereas Point's normalization uses [-180,180].  Almost certainly, either will work.

Reply:
I converted the (lat,lon) pairs to Points and the issue disappears.